lib/internal/validators.js

```diff
@@ -39,7 +39,11 @@ const modeDesc = 'must be a 32-bit unsigned integer or an octal string';
  * @param {number} def If specified, will be returned for invalid values
  * @returns {number}
  */
-function parseMode(value, name, def) {
+function parsePermissionMode(value, name, def) {
```

Comment:
File modes do not just encode permissions. If you want to rename this, I’d suggest `parseFileMode`.